styles/info.scss

```diff
@@ -64,45 +69,58 @@
             color: $link-hovered-color;
           }
         }
-      }
 
-      .location-icons {
-        display: inline-flex;
-        width: $icon-dim;
-        height: $icon-dim;
+        .location-icons {
+          display: inline-flex;
+          width: $icon-dim;
+          height: $icon-dim;
 
-        justify-content: center;
-        align-items: center;
+          margin-right: $spc-s;
 
-        font-size: 1.625rem;
-        color: $link-hovered-color;
-        border-radius: 100%;
-        box-shadow: $icon-shadow $text-shadow-color;
+          justify-content: center;
+          align-items: center;
+          text-align: center;
+
+          font-size: 1.625rem;
+          color: $link-hovered-color;
+          border-radius: 100%;
+          box-shadow: $icon-shadow $text-shadow-color;
```

Comment:
`$icon-shadow` seems to be holding only the dimension of the shadow, let's name it in a way that makes it obvious.